File: School/Chapter3/factorial.js
// let number = 7

function factorial(number) {
    let counter = 1
    let factorial = 1

    // while (counter <= number) {
    //     factorial = factorial * counter
    //     counter++
    // }

    // do {
    //      factorial = factorial * counter
    //      counter++
    // } while (counter<= number);
    for  (let index = 1; index <= number; index++) {
        factorial = factorial * index
    }
    console.log(factorial)
}

factorial(5)
factorial(6)
factorial(3)
factorial(2)

File: 100L/School/Chapter3/factorial.c
#include <stdio.h>

int main()
{
    unsigned int number, counter, factorial;
    printf("Enter the number: ");
    scanf("%d", &number);
    counter = 1;
    factorial = 1;
    while (counter <= number)
    {
        factorial = factorial * counter;
        counter++;
    }

    printf("%d", factorial);
    return 0;
}
